index.src.html

```diff
@@ -769,103 +782,123 @@ <h3 id=interface-reporting-observer>Interface {{ReportingObserver}}</h3>
 
 [NoInterfaceObject]
 interface Report {
-    readonly attribute DOMString type;
-    readonly attribute DOMString url;
-    readonly attribute ReportBody? body;
+  readonly attribute DOMString type;
+  readonly attribute DOMString url;
+  readonly attribute ReportBody? body;
 };
 
 [Constructor(ReportingObserverCallback callback)]
 interface ReportingObserver {
-    void observe(optional ReportingObserverOptions options);
-    void disconnect();
+  void observe(optional ReportingObserverOptions options);
+  void disconnect();
 };
 
 callback ReportingObserverCallback = void (sequence&lt;Report> reports, ReportingObserver observer);
 
 dictionary ReportingObserverOptions {
   sequence&lt;DOMString> types;
+  boolean futureOnly = false;
 };
   </pre>
-  A {{ReportingObserver}} object can be used to observe some types of
-  <a>reports</a> from JavaScript when they are created.
-  
+
   Issue: How do we specify which reports are observable?  Network-layer reports
   like CSP reports will not be.  Maybe there's something about the settings
   object?
-  
+
   A <dfn id=dom-report interface>Report</dfn> is the application exposed
   representation of a <a>report</a>. <dfn attribute for="Report">type</dfn>
   returns [=report/type=], <dfn attribute for="Report">url</dfn> returns
-  [=report/url=] and <dfn attribute for="Report">body</dfn> returns
+  [=report/url=], and <dfn attribute for="Report">body</dfn> returns
   [=report/body=].
-    
+
   Each {{ReportingObserver}} object has these associated concepts:
-    - A <dfn for=ReportingObserver>callback</dfn> set on creation.
-    - An {{ReportingObserverOptions}} dictionary called
+    - A <dfn for=ReportingObserver>callback</dfn> function set on creation.
+    - A {{ReportingObserverOptions}} dictionary called
       <dfn for=ReportingObserver>options</dfn>.
     - A list of {{Report}} objects called the <dfn for=ReportingObserver>report
-      list</dfn> that is initially empty.
+      list</dfn>, which is initially empty.
 
   The <dfn constructor for=ReportingObserver><code>
   ReportingObserver(|callback|)</code></dfn> constructor, when invoked, must
   create a new {{ReportingObserver}} object with <a>callback</a> set to
   |callback|, and then return it.
```

Comment:
Yes. Actually, I would like to have the options set in the constructor rather than in observe(), they will now be initialized to the passed in parameter.